### DISCUSSION TITLE
Dedicated page to list all talks

### DISCUSSION THREAD

**Discussion by WengerK** [webmardi/webmardi.ch#47] [Ideas]
Hello @Yago 

What do you think about the following idea **Having a dedicated page to list all talks** instead of the current infinity click/scroll ?

I think it's a good idea because:
- For now the talks does not have the year, meaning we don't really know when the talk has been given. I think with a dedicated page, we could create separator (like the heading "Past" and "Upcoming") by year.
- It would be easier to search for a talk in the list (without having to "click").
- It would be better for seo (maybe?)

**Comment by Yago** [comment]
I'm personally not a big fan of infinite scroll, so yes, it's not a bad idea 😉   Do you prefer an `/events` page with all the events and no infinite click-scroll or a `/past-events` page in the same spirit, but without the upcoming events?

Here are some ideas of how it might look:

![Webmardi - Monthly events in western Switzerland](https://user-images.githubusercontent.com/2025924/143087665-bf3e8c42-a01f-4377-94c1-074e6fb975af.png)

![Webmardi - Monthly events in western Switzerland](https://user-images.githubusercontent.com/2025924/143087685-c8667bab-9b5f-4485-88f4-b927df1a20b2.png)

**Reply by WengerK** [reply]
I love the idea of having all the talks (upcoming and past ones).

One thing I would change on the proposition. Instead of upcoming/past can we separate talks by years (meaning we will have more separator that the 2 we currently have). ?

**Reply by Yago** [reply]
I love this idea of year separation too ! I make a lot of sens on this page.

**Comment by WengerK** [comment]
May we go further by creating an issue in order to follow progress on this task ?

**Reply by Yago** [reply]
I just [release](https://github.com/webmardi/webmardi.ch/releases/tag/v1.1.1) a first version of this idea 🎉

**Comment by WengerK** [comment]
YAAAAY that's awesome !!! I will present it to the team Wednesday :) 

MANY MANY Thanks <3
